DemoServer/Controllers/Demos/Attachments/IndexAttachmentDetails/metadata.json

```diff
@@ -0,0 +1,160 @@
+                }
+            ]
+        },
+        {
+            "Title": "Load details for indexing",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* In the index Map function, call `AttachmentsFor()` to load the document's attachment details to be indexed.",
+                "",
+                "* The _attachments_ array will hold the following per attachment:",
+                "  * Name",
+                "  * Type",
+                "  * Hash",
+                "  * Size",
+                "",
+                "* To index an attachment content use: _LoadAttachment()_ or _LoadAttachments()_."
+            ],
+            "Assets": [
+                {
+                    "Title": "using AttachmentsFor",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/document-extensions/attachments/indexing#using-attachmentsfor"
+                },
+                {
+                    "Title": "using LoadAttachment",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/document-extensions/attachments/indexing#indexes-with--1"
+                },
+                {
+                    "Title": "using LoadAttachments",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/document-extensions/attachments/indexing#indexes-with--2"
+                }
+            ]
+        },
+        {
+            "Title": "Configure index fields",
+            "Slug" : "step-4",
+            "DescriptionLines": [
+                "* Configure the index-fields content.",
+                "",
+                "* The attachments names, types, hashes & sizes are now available and can be used to configure the index fields.<br>",
+                "  e.g., _AttachmentNames_ is an index field that holds all the attachments names of the document.",
+                "",
+                "* Later on, the 'Employees' collection can be queried using this index to get employees that have specific attachments names, types, etc.",
+                "",
+                "* Re-indexing will happen on any change to the document's attachments."
+            ],
+            "Assets": [
+                {
+                    "Title": "Indexing Attachments",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/document-extensions/attachments/indexing"
+                }
+            ]
+        },
+        {
+            "Title": "Query for documents",
+            "Slug" : "step-5",
+            "DescriptionLines": [
+                "* Query the 'Employees' collection using the defined index.",
+                "",
+                "* The query results will include all documents from the 'Employees' collection that have attachments with a matching Type and Size."
```

Comment:
taking into account query is 'employee.AttachmentSizes.Any(x => x > attachmentSize)' would it be appropriate to state "minimum Size"?

Reply:
done